### PR TITLE
Summer updates

### PR DESCRIPTION
**Pull request by Roleren**
Speedup, fixed bugs, updated docs

### CODE REVIEW

File: NAMESPACE [hunk 1]
```diff
@@ -51,6 +51,7 @@ export(tile1)
 export(translationalEff)
 export(txNames)
 export(txNamesWithLeaders)
+export(txSeqsFromFa)
```
Comment (JokingHero) [inline NAMESPACE]: completely unnecessary export? there exist function readDNAStringSet

File: R/feature_helpers.R [hunk 2]
```diff
@@ -41,56 +41,51 @@ txLen <- function(Gtf = NULL, changedFiveUTRs = NULL){
 #' Get hits per codon
 #'
 #' Helper for entropy function, normally not used directly
-#' @param h indices per tuple
-#' @param indeces whole list of indices
-#' @param L Lengths
-#' @param N hit sums
-#' @param reg_len size of runs
-#' @param runLengths integers per run
+#' Seperate each group into tuplets (codons)
```
Comment (JokingHero) [inline R/feature_helpers.R]: triplets?

File: R/feature_helpers.R [hunk 3]
```diff
@@ -41,56 +41,51 @@ txLen <- function(Gtf = NULL, changedFiveUTRs = NULL){
...
-
-  if (length(L) > 1) { # if more than 1 hit total
-    acums <- cumsum(as.numeric(L))
-    unlacums <- unlist(lapply(seq(length(runLengths)-1), function(x) {
-      rep(acums[x], runLengths[x+1])
-    }), use.names = FALSE)
-    unlacums <- unlist(c(rep(1, runLengths[1]), unlacums))
+codonSumsPerGroup <- function(countList, reg_len,
+                              runLengths ){
+  # TODO: Check behavior on groups with lengths that are prime
```
Comment (JokingHero) [inline R/feature_helpers.R]: if you have suspicion than why not write a simple test?

File: NAMESPACE (hunk 1 above)
Comment (Roleren) [reply NAMESPACE]: Okey to keep this then ?

**Pull request merged**